client/src/components/addSighting/addSighting.jsx

```diff
@@ -71,6 +72,10 @@ export class NewSighting extends React.Component {
 				{/* <GoogleMap view="create" onClick={this.handleLocationChange} /> */}
 				<div id="addSightingContainer">
 					<form id="addSightingForm" onSubmit={this.handleSubmit}>
+						<div id="locationDevice">
+							Click map to select location or check to use device location {/*Currently will only use map location*/}
```

Comment:
Why not just have device location automatically put a pin down for you? Then its always simply tap to adjust